test/metabase/api/activity_test.clj

```diff
@@ -22,19 +22,19 @@
 ;; NOTE: timestamp matching was being a real PITA so I cheated a bit.  ideally we'd fix that
 (tt/expect-with-temp [Activity [activity1 {:topic     "install"
                                            :details   {}
-                                           :timestamp (u/->Timestamp "2015-09-09T12:13:14.888Z")}]
+                                           :timestamp (u/->Timestamp #inst "2015-09-09T12:13:14.888Z")}]
                       Activity [activity2 {:topic     "dashboard-create"
                                            :user_id   (user->id :crowberto)
                                            :model     "dashboard"
                                            :model_id  1234
                                            :details   {:description  "Because I can!"
                                                        :name         "Bwahahaha"}
-                                           :timestamp (u/->Timestamp "2015-09-10T18:53:01.632Z")}]
+                                           :timestamp (u/->Timestamp #inst "2015-09-10T18:53:01.632Z")}]
                       Activity [activity3 {:topic     "user-joined"
                                            :user_id   (user->id :rasta)
                                            :model     "user"
                                            :details   {}
-                                           :timestamp (u/->Timestamp "2015-09-10T05:33:43.641Z")}]]
+                                           :timestamp (u/->Timestamp #inst "2015-09-10T05:33:43.641Z")}]]
```

Comment:
so `->Timestamp` doesn't work on strings anymore? The idea of that function was to take anything that could conceivably represent a timestamp and turn it into a `Timestamp` object... ISO-8601 Strings represent datetimes...